gap/profiling.gi

```diff
@@ -760,6 +761,14 @@ function(data, outfile, jobid, pathtoremove)
         return SplitString(str, " ")[1];
     end;
 
+    # GAP's ReplacedString does not terminate for
+    # empty string to replace
```

Comment:
Probably should be a bug report for GAP, too? (perhaps you already made one, I did not check)

Reply:
I have indeed: https://github.com/gap-system/gap/issues/3134